inst/preview/index.html

```diff
@@ -52,14 +52,22 @@ <h3>Learn more about this pin at:</h3>
     </section>
 
     <section>
-    <h3>R Code</h3>
-      <pre id="pin-r" class="pin-code"><code class="r">library(pins)
+      <h3>R Code</h3>
+        <pre id="pin-r" class="pin-code"><code class="r">library(pins)
 board <- {{{board_deparse}}}
 pin_read(board, "{{pin_name}}")</code></pre>
-    <script type="text/javascript">
-      hljs.registerLanguage("r", highlight_r);
-      hljs.initHighlightingOnLoad();
-    </script>
+      <script type="text/javascript">
+        hljs.registerLanguage("r", highlight_r);
+        hljs.registerLanguage("python", highlight_python);
+        hljs.initHighlightingOnLoad();
+      </script>
+    </section>
+    
+    <section style="{{show_python_style}}">
+      <h3>Python Code</h3>
+        <pre id="pin-python" class="pin-code"><code class="python">import pins
+board = pins.board_connect()
+board.pin_read(board, "{{pin_name}}")</code></pre>
```

Comment:
Is this the best option for Python code? I believe this is what people should use if they are publishing code to Connect, right? Because the env vars are provided?

Reply:
you'll need to bring the environment variables into your session somehow, but once they are there, you won't have to add any special arguments. something like this would work ⏬ 

```
import pins
from dotenv import load_dotenv
load_dotenv()

board = pins.board_connect()
```